modules/get-started/pages/install-beta.adoc

```diff
@@ -1,7 +1,7 @@
 = Install 23.3 Beta
 :description: Learn how to install the Beta version. 
 
-Redpanda Beta versions provide users the opportunity to test and share feedback on new features before they're finalized in general availability. Beta versions, like `v23.3.1-rc1`, are published to `redpanda-unstable` as release candidate (RC) builds. RC builds are not recommended for production use.
+Redpanda Beta versions provide users the opportunity to test and share feedback on new features before they're finalized in general availability. Beta versions, like `v24.1.1-rc4`, are published to `redpanda-unstable` as release candidate (RC) builds. RC builds are not recommended for production use.
```

Comment:
I think we’ve started using lowercase ‘beta’ now. Please check for other instances.